constraint/pkg/types/validation.go

```diff
@@ -9,6 +9,27 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
+type EvaluationEngineType string
+
+const (
+	Undefined  EvaluationEngineType = "undefined"
+	RegoEngine EvaluationEngineType = "rego"
+)
+
+type ReviewMeta struct {
+	// EvaluationLatency is the number of milliseconds it took to server a client.Review() call.
+	EvaluationLatency float64
```

Comment:
At first glance, latency and batch size can be easily determined by a consumer of the API call, like the gatekeeper at the call site.

But the underlying measurement here is the evaluation time during the engine call (`driver.Query()`)., which is what we are trying to surface. This way, future iterations of the `Review` API have a notion of the time spent under evaluation for a `review` object and not how long the `Review()` func took to work/ execute overall.